### PR TITLE
BaSP-TG-61: Select input component

### PR DESCRIPTION
**Pull request by LucianoFranchino**
Shared component created: Select input

- Added in Timesheets form
- Added in Project form

### CODE REVIEW

File: src/Components/TimeSheets/Form/timesheetForm.js [hunk 1]
```diff
@@ -148,26 +149,54 @@ const Form = () => {
...
+              data={employees.map((employee) =>
+                !employee
+                  ? ''
+                  : {
+                      id: employee._id,
+                      value: employee.name
+                    }
+              )}
+            />
+            {/* // <label htmlFor="project">Project</label>
```
Comment (lmalvasia) [inline src/Components/TimeSheets/Form/timesheetForm.js:180]: We should remove all of the commented code.

File: src/Components/TimeSheets/Form/timesheetForm.js [hunk 2]
```diff
@@ -148,26 +149,54 @@ const Form = () => {
...
+            //       <option key={project._id} value={project._id}>
+            //         {project.name}
+            //       </option>
+            //     );
+            //   })}
+            // </select> */}
           </div>
           <div>
-            <label htmlFor="employee">Employee</label>
+            {/* <label htmlFor="employee">Employee</label>
```
Comment (lmalvasia) [inline src/Components/TimeSheets/Form/timesheetForm.js:199]: We should remove all of the commented code.

File: src/Components/TimeSheets/Form/timesheetForm.js [hunk 3]
```diff
@@ -183,10 +212,24 @@ const Form = () => {
...
+              data={tasks.map((task) =>
+                !task
+                  ? ''
+                  : {
+                      id: task._id,
+                      value: task.description
+                    }
+              )}
+            />
+            {/* <label htmlFor="task">Task</label>
```
Comment (lmalvasia) [inline src/Components/TimeSheets/Form/timesheetForm.js:232]: We should remove all of the commented code.

File: src/Components/TimeSheets/Form/timesheetForm.js (hunk 1 above)
Comment (LucianoFranchino) [reply src/Components/TimeSheets/Form/timesheetForm.js:180]: Resolved

File: src/Components/TimeSheets/Form/timesheetForm.js (hunk 2 above)
Comment (LucianoFranchino) [reply src/Components/TimeSheets/Form/timesheetForm.js:199]: Resolved

File: src/Components/TimeSheets/Form/timesheetForm.js (hunk 3 above)
Comment (LucianoFranchino) [reply src/Components/TimeSheets/Form/timesheetForm.js:232]: Resolved

File: src/Components/Projects/Form/index.js [hunk 4]
```diff
@@ -203,50 +204,43 @@ const Form = () => {
...
+                { id: 'DEV', value: 'DEV' },
+                { id: 'TL', value: 'TL' },
+                { id: 'PM', value: 'PM' },
+                { id: 'QA', value: 'QA' }
+              ]}
+            />
+            <label>Employees</label>
+            <SelectInput
+              name="employe"
+              label="Employe"
```
Comment (GinaSchP) [inline src/Components/Projects/Form/index.js:227]:
```suggestion
              label="Employee"
```

File: src/Components/Projects/Form/index.js [hunk 5]
```diff
@@ -203,50 +204,43 @@ const Form = () => {
...
+              data={[
+                { id: 'DEV', value: 'DEV' },
+                { id: 'TL', value: 'TL' },
+                { id: 'PM', value: 'PM' },
+                { id: 'QA', value: 'QA' }
+              ]}
+            />
+            <label>Employees</label>
+            <SelectInput
+              name="employe"
```
[the remark this answers is not in the document]
Comment (LucianoFranchino) [reply src/Components/Projects/Form/index.js]: Resolved

File: src/Components/Projects/Form/index.js (hunk 4 above)
Comment (LucianoFranchino) [reply src/Components/Projects/Form/index.js]: Resolved

**Review by lmalvasia — APPROVED** [review]

**Pull request merged**